app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.name = name
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(human["name"], human["age"]) for human in people]
+    name_person = {person.name: person for person in result}
+    for i, human in enumerate(people):
+        if "wife" in human and human["wife"] is not None:
+            result[i].wife = name_person.get(human["wife"])
+        if "husband" in human and human["husband"] is not None:
+            result[i].husband = name_person.get(human["husband"])
```

Comment:
The code assumes that `wife` and `husband` attributes exist on `Person` objects, but these attributes are not defined in the `Person` class. This will lead to an `AttributeError` if these attributes are accessed. Consider adding these attributes to the `Person` class.